3rdparty/CMakeLists.txt

```diff
@@ -1,7 +1,4 @@
-# Version 1.4.4
+# Vc version 1.4.4
 add_subdirectory(Vc)
-set(conky_libs ${conky_libs} Vc)
-set(conky_includes ${conky_includes} "${CMAKE_CURRENT_SOURCE_DIR}/Vc")
 
-set(conky_libs ${conky_libs} PARENT_SCOPE)
-set(conky_includes ${conky_includes} PARENT_SCOPE)
```

Comment:
Why not keep 3rdparty related build configuration in 3rdparty/CMakeLists.txt?

Reply:
I dislike this because it feels like the dependency is going in the wrong direction (from lower -> upper scope). The child shouldn't know about the parent (i.e., you should be able to take the code out of `3rdparty/` and move it elsewhere and it should work all the same).